src/features/movies/MoviePage/index.js

```diff
@@ -42,8 +44,15 @@ export default () => {
     }
   }, [query, history, searchParams]);
 
+  if(error)
+  {
+    return  <Wrapper> 
+              <Error/>
+            </Wrapper> 
+  }
+
   return (
-    !loading && movie ?
+    !loading && movie && movie.credits.cast ?
```

Comment:
dodałem taki warunek bo próbowało renderować przy wpisaniu randomowego url'a - szczerze to nie wiem dlaczego